src/pages/sign-up/components/signUpForm.ts

```diff
@@ -0,0 +1,124 @@
+
+import compileTemplate from './signUpForm.pug';
+import './signUpForm.css';
+import { Form, IFormInput } from '../../../utils/form';
+import { Button } from '../../../components/button/button';
+import { Link } from '../../../components/link/link';
+
+interface IFormProps {
+    title: string;
+    events?: any
+}
+
+const formInputs: Record<string, IFormInput> = {
+    'loginInput': {
+        props: {
+            placeholder: 'Логин',
+            inputType: 'text',
+            name: 'login',
+            wrapperClass: 'form__text-input-wrapper',
+            error: '',
+            inputId: 'loginInput',
+            value: ''
```

Comment:
Последние строки в таких местах лучше тоже заканчивать на `,` - это положительно сказывается на диффах в система контроля версий
```suggestion
            value: '',
```